tests/h/views/activity_test.py

```diff
@@ -181,19 +181,13 @@ def test_search_returns_group_info_if_user_a_member_of_group(self,
         assert group_info['name'] == group.name
         assert group_info['pubid'] == group.pubid
 
-    def test_search_checks_whether_the_user_has_admin_permission_on_the_group(
-            self, controller, group, pyramid_request):
-        pyramid_request.authenticated_user = group.members[-1]
-
-        controller.search()
-
-        pyramid_request.has_permission.assert_called_once_with('admin', group)
-
     def test_search_does_not_show_the_edit_link_to_group_members(self,
                                                                  controller,
                                                                  group,
                                                                  pyramid_request):
-        pyramid_request.has_permission = mock.Mock(return_value=False)
+        def fake_has_permission(permission, context=None):
+            return permission != 'admin'
+        pyramid_request.has_permission = mock.Mock(side_effect=fake_has_permission)
```

Comment:
In an ideal world, we wouldn't mock out has_permission here, we'd provide a stub context object with an appropriate ACL and let the view function worry about how to test that.

Just a comment -- no need to change anything.